generator/src/main/java/com/scottlogic/deg/generator/inputs/RuleInformation.java

```diff
@@ -1,15 +1,16 @@
 package com.scottlogic.deg.generator.inputs;
 
-import com.scottlogic.deg.schemas.v0_1.RuleDTO;
-
 import java.util.Objects;
 
 public class RuleInformation {
     private final String description;
 
-    public RuleInformation(RuleDTO rule) {
-        String ruleDescription = rule != null ? rule.rule : null;
-        this.description = ruleDescription != null ? ruleDescription : "Unnamed rule";
+    public RuleInformation(){
+        description = "Unnamed rule";
```

Comment:
To be picky, I would use `public RuleInformation() { this(null); }` to avoid defining your dummy constant string in two places.

Reply:
done